benchmark/benchmarks/transfer/transfer.go

```diff
@@ -407,21 +413,14 @@ func Init(cmd *cobra.Command) {
 		viper.BindPFlag(v, cmd.Flags().Lookup(v)) // nolint: errcheck
 	}
 
-	// TODO: This probably shouldn't be hardcoded, but just initialize
-	// the funding account here for now.
-	const (
-		fundingAccount = "533d62aea9bbcb821dfdda14966bb01bfbbb53b7e9f5f0d69b8326e052e3450c"
-		fundingAddress = "0x7110316b618d20d0c44728ac2a3d683536ea682b"
-	)
-
-	privKey, err := crypto.HexToECDSA(fundingAccount)
+	privKey, err := crypto.HexToECDSA(flagFundingAccount)
 	if err != nil {
-		panic(err)
+		panic(fmt.Sprintf("error parsing funding account private key: %v", err))
 	}
 
 	// Sanity check the addresss.
 	addr := privKeyToAddress(privKey)
-	if strings.ToLower(addr.Hex()) != fundingAddress {
+	if strings.ToLower(addr.Hex()) != flagFundingAccountAddress {
```

Comment:
```suggestion
	if strings.ToLower(addr.Hex()) != strings.ToLower(flagFundingAccountAddress) {
```
there's no guarantee that the user won't pass in a lower-cased address